scripts/build/architecture-convert.sh

```diff
@@ -51,17 +51,25 @@ if [ "$ARGV_TYPE" == "node" ]; then
     RESULT=ia32
   elif [ "$ARGV_ARCHITECTURE" == "x64" ]; then
     RESULT=x64
-  elif [ "$ARGV_ARCHITECTURE" == "armv7l" ]; then
+  elif [ "$ARGV_ARCHITECTURE" == "armhf" ]; then
```

Comment:
It might also be worth this script checking that the supplied `$ARGV_ARCHITECTURE` is in the list `[x86, x64, armv7hf]` of 'recognised arches' before then checking the `$ARGV_TYPE`?